tests/test_idutils.py

```diff
@@ -774,7 +776,8 @@
 ]
 
 
-def test_detect_schemes():
+@pytest.fixture(scope="session")
```

Comment:
Sorry for dropping-in on this PR! It's a module we rely on, so I get notifications for it and I didn't see anyone mention this: why were these `pytest.fixture`s added? Those are tests (not fixtures), so apriori there shouldn't be fixture decorators there :thinking: . Thanks!

Reply:
Thanks for pointing it out, I was trying different things and seem to have left it there. Have removed it now!